lib/Analysis/RotationAnalysis/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+add_mlir_dialect_library(MLIRHEIRAnalysisROT
```

Comment:
I'm not sure if `add_mlir_dialect_library` is the right thing to use here, given that this is an analysis pass? 
According to https://mlir.llvm.org/docs/Tutorials/CreatingADialect/ it should be `add_mlir_conversion_library`?

Also, I think it'd be nice to keep the names of targets consistent across Bazel and CMake (modulo some prefix for CMake), e.g., `HEIRRotationAnalysis` for this target.

Reply:
TBH there is over 60+ files linked against the heir-opt binary; otherwise its plain and simple.

Reply:
done